demo/manage.py

```diff
@@ -7,16 +7,10 @@
     try:
         from django.core.management import execute_from_command_line
     except ImportError:
-        # The above import may fail for some other reason. Ensure that the
-        # issue is really that Django is missing to avoid masking other
-        # exceptions on Python 2.
```

Comment:
I'm not sure this is relevant anymore? Might be good to update default project as well

Reply:
this is why I usually just use `django-admin` in libraries, updating `manage.py` is annoying. Feel free to update default project with the upstream manage.py but we have some local modifications to keep in mind.